Toggl.Droid/Autocomplete/AutocompleteExtensions.cs

```diff
@@ -19,6 +19,17 @@ public static class AutocompleteExtensions
         public static IImmutableList<ISpan> AsImmutableSpans(this ICharSequence text, int cursorPosition)
             => text.AsSpans(cursorPosition).ToImmutableList();
 
+        public static bool isEqualTo(this ISpannable first, ISpannable second)
+        {
+            if (first.ToString() != second.ToString()) return false;
```

Comment:
control-flow blocks should go to a new line
```suggestion
            if (first.ToString() != second.ToString()) 
                return false;
```